drawingutility.cpp

```diff
@@ -0,0 +1 @@
+#include "drawingutility.hpp"
```

Comment:
Mmmmmmmmm I'm not sure what's going on here. Is this final? If it isn't, idk why this was setup for a PR